app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -63,9 +62,6 @@ import javax.inject.Named
 
 class EditPatientScreen(context: Context, attributeSet: AttributeSet) : RelativeLayout(context, attributeSet), EditPatientUi {
 
-  @Inject
-  lateinit var controller: PatientEditScreenController
```

Comment:
Wasn't this being used in the class? 😅

Reply:
Haha... it wasn't! Downside of copy-paste.